all-actors/src/main/java/org/sunbird/utilities/CertificateUtil.java

```diff
@@ -35,9 +54,31 @@ public static boolean isIdPresent(String certificateId) {
     }
 
 
-    public static void insertRecord(Map<String,Object>certAddReqMap){
-        ElasticSearchHelper.getResponseFromFuture(elasticSearchService.save(JsonKeys.CERT,(String)certAddReqMap.get(JsonKeys.ID),certAddReqMap));
+    public static Response insertRecord(Map<String,Object>certAddReqMap) throws BaseException {
+        Map<String,Object>certMap = new HashMap<>();
+        long createdAt = System.currentTimeMillis();
+        certAddReqMap.put(JsonKeys.CREATED_AT,createdAt);
```

Comment:
In Es createdAt will save as long value only right? while in cassandra it is timestamp only right?

Reply:
Yes, Thats why i have two map certAddReqMap for ES, and certMap for cassandra